tests/test_core.py

```diff
@@ -265,3 +265,15 @@ def test_add_metadata_as_int(meta_df):
 
     obs = meta_df['meta_int']
     pd.testing.assert_series_equal(obs, exp)
+
+
+def test_filter_by_metadata_bool(meta_df):
+    meta_df.metadata([True, False], 'exclude')
```

Comment:
pedantic, I know, but can you add the kwarg `name='exclude'` please? I was curious what this was and went searching..